lib/najax.js

```diff
@@ -138,7 +138,11 @@ function najax (uri, options, callback) {
         try {
           data = JSON.parse(data.replace(/[\cA-\cZ]/gi, ''))
         } catch (e) {
-          return onError(e)
+          //emulate jQuery functionality (replace data with JSON parse error)
```

Comment:
Ugh.. this whole time I was misreading the error and thought it was talking about the test-najax.js file.